iphone/Maps/Classes/CustomAlert/Toast/Toast.swift

```diff
@@ -72,15 +77,22 @@ final class Toast: NSObject {
       self.blurView.alpha = 1
     }
 
-    Timer.scheduledTimer(timeInterval: 3,
-                         target: self,
-                         selector: #selector(onTimer),
-                         userInfo: nil,
-                         repeats: false)
+    timer = Timer.scheduledTimer(timeInterval: 3,
+                                 target: self,
+                                 selector: #selector(onTimer),
```

Comment:
It seems like the `onTimer()` method can be removed now and you can pass the `hide` in selector.